.travis.yml

```diff
@@ -132,6 +132,11 @@ script:
           cd for_testing
           cp ../.coveragerc .
           nosetests --with-doctest --with-coverage --cover-package nibabel nibabel
+          pytest -v ../nibabel/tests/test_a*.py ../nibabel/tests/test_b*.py ../nibabel/tests/test_c*.py ../nibabel/tests/test_d*.py
+          pytest -v../nibabel/tests/test_e*.py ../nibabel/tests/test_f*.py ../nibabel/tests/test_h*.py
+          pytest -v../nibabel/tests/test_i*.py ../nibabel/tests/test_k*.py ../nibabel/tests/test_l*.py ../nibabel/tests/test_m*.py
+          pytest -v../nibabel/tests/test_n*.py ../nibabel/tests/test_o*.py ../nibabel/tests/test_p*.py ../nibabel/tests/test_q*.py
+          pytest -v ../nibabel/tests/test_w*.py
           pytest -v ../nibabel/tests/test_affines.py ../nibabel/tests/test_volumeutils.py
```

Comment:
To test the installed version of nibabel, we can use:

```
pytest --pyargs nibabel
```

See https://docs.pytest.org/en/latest/goodpractices.html#tests-as-part-of-application-code